onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.h

```diff
@@ -105,16 +105,16 @@ class ModelBuilder {
 
   int32_t GetEffectiveFeatureLevel() const { return nnapi_effective_feature_level_; }
 
-  // Just for Debugging
-  void SetDebugTrackNode(const size_t node_index) {
 #ifndef NDEBUG
+  // Set the node index to be tracked
+  // For now, NNAPI is a black box, LOGs are limited and this EP is hard to debug.
+  // ONNX node and NNAPI node are not a 1:1 mapping, like batch-normalization.
+  // We use this to track a specific ONNX node which we are processing and record the detail mapping relationship.
+  // So we can log out each NNAPI OP status during model-building and compiling.
+  void SetDebugTrackNode(const size_t node_index) {
```

Comment:
ok, thanks for adding some comments here.
I'd also suggest a more specific name. maybe something like SetDebugCurrentOnnxNodeIndex() and current_onnx_node_index_

Reply:
Sure